inginious/frontend/plugins/show_hints/pages/hints_edit.py

```diff
@@ -0,0 +1,80 @@
+import os
+from collections import OrderedDict
+from inginious.frontend.pages.course_admin.task_edit import CourseEditTask
+from .constants import use_minified
+
+_SHOW_HINTS_TEMPLATES_PATH = os.path.join(os.path.dirname(__file__),'templates')
+
+def edit_hints_tab(course, taskid, task_data, template_helper):
+
+    tab_id = 'hints'
+    link = '<i class="fa fa-question"></i>&nbsp; ' + _('Hints')
+
+    add_static_files(template_helper)
+
+    #Get task data
+    task_hints = task_data.get("task_hints")
```

Comment:
Shouldn't this have a default value? like an empty list. Not all tasks have that key in the data.